### PR TITLE
Feature/improve generate

### PR DESCRIPTION
**Pull request by JohannesRudolph**

### CODE REVIEW

**Review by DorukAkinci — APPROVED** [review]

File: unipipe/commands/generate.ts [hunk 1]
```diff
@@ -64,9 +64,27 @@ async function generateUniPipeDeployment() {
...
       );
       break;
-    case "aci_tf":
-      console.log(unipipeOsbAciTerraform);
+    case "aci_tf": {
+      const tfPath = await Input.prompt({
+        message:
+          "Pick a destination directory for the generated terraform files:",
+        default: "./",
+      });
```
Comment (DorukAkinci) [inline unipipe/commands/generate.ts]: I believe we should put this `Input.prompt` part on top of the Switch-case section. All of our `Cases` will use the same Input structure

File: unipipe/commands/generate.ts [hunk 2]
```diff
@@ -64,9 +64,27 @@ async function generateUniPipeDeployment() {
...
+
+      const mainTf = path.join(tfPath, "main.tf");
+      await Deno.writeTextFile(
+        mainTf,
+        unipipeOsbAciTerraform,
+      );
```
Comment (DorukAkinci) [inline unipipe/commands/generate.ts]: I have figured out that this function doesnt handle the non-existing folders. i believe we should handle it as well and make it separated function. maybe we can create smth like `await writeFile(path, terraform_template)` again but this time our function will also create the whole folder structure if it is necessary.

Comment (JohannesRudolph) [reply unipipe/commands/generate.ts]: I've refactored this to write using the dir utils that we have, as they handle all of that functionality gracefully already

**Comment by DorukAkinci** [comment]
I have changed the code a little bit because we were talking about creating 2 different terraform folders for the GCP Cloud Run template, so we dont use the true/false boolean method like we are doing at the moment. So with the latest changes i'm thinking like i can chain these multiple instruction outputs and i can even change the initial `Instructions:` texts, so it can look better and more smoother.

**Pull request merged**